package.json

```diff
@@ -10,6 +10,11 @@
     "base64",
     "gzip"
   ],
+  "dependencies": {
+    "stream":"~0.0.2",
+    "util": "~0.10.3",
+    "string_decoder": "~0.10.31"
```

Comment:
these still do exist as npm modules & including them explicitly makes sure you're not relying on the version of node running the code.